src/http.cpp

```diff
@@ -44,10 +44,6 @@
 #define SLEEP 1000
 #define SNOOZE 0
 
-#ifdef __MINGW__
-#define fopen_S(f, n, a) f = fopen(n, a)
```

Comment:
I was wondering why this dissapeared and after checking the code I noticed that it is not used at all. Is it something that clang-tidy deleted automatically or was it a manual change?

Reply:
It was manual. However clang-tidy does warn about this in one of its checks. It recommends doing away with function macros and using constexpr functions (mostly the same behavior).